errors.md

```diff
@@ -38,6 +38,8 @@
 | 1157 | error getting catalog item API server info properties                                                       | pkg/apic/ErrGetCatalogItemServerInfoProperties      |
 | 1158 | subscription manager is not in a running state                                                              | pkg/apic/ErrSubscriptionManagerDown                 |
 | 1160 | error getting endpoints for the API specification                                                           | pkg/apic/ErrSetSpecEndPoints                        |
+| 1161 | error deleting API Service for catalog item in AMPLIFY Central                                              | pkg/agent/ErrDeletingService                         |
+| 1162 | error deleting catalog item in AMPLIFY Central                                                              | pkg/agent/ErrDeletingCatalogItem                     |
```

Comment:
Since updating everywhere else, Amplify?

Reply:
Found 10 others in there, also. Fixed.